client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -27,6 +27,23 @@ function MetadataCtrl($scope, desks, metadata, $filter, privileges, adminPublish
         });
     };
 
+    
+    $scope.$watch('item.publish_schedule_date', function() {
+        setPublishScheduleDate();
+    });
+
+    $scope.$watch('item.publish_schedule_time', function() {
+        setPublishScheduleDate();
+    });
+
+    function setPublishScheduleDate() {
+        if ($scope.item.publish_schedule_date && $scope.item.publish_schedule_time) {
+            $scope.item.publish_schedule = datetimeHelper.mergeDateTime($scope.item.publish_schedule_date,
+                $scope.item.publish_schedule_time).format();
+            $scope.autosave($scope.item);
```

Comment:
I think this is the way to set dirty flag so that save button gets enabled. At least this is how I see it.